src/Symfony/Component/Notifier/Bridge/Esendex/.gitattributes

```diff
@@ -0,0 +1,4 @@
+/Tests export-ignore
+/phpunit.xml.dist export-ignore
+/.gitattributes export-ignore
```

Comment:
Could be removed as it's not in the other .giattributes files (gains are insignificant anyway).